packages/framework/fluid-framework/src/containerDefinitions.ts

```diff
@@ -3,4 +3,8 @@
  * Licensed under the MIT License.
  */
 
-export { AttachState } from "@fluidframework/container-definitions";
+export {
+    AttachState,
```

Comment:
This should be unified * exports, but it looks like we have some conflicts across container-definitions / driver-definitions.

Reply:
We're switching to explicit exports across the repo anyway.

Reply:
The fluid-framework is somewhat specific is all that it does is re-exports. We should reevaluate if this is the best way forward.